@ecomplus/storefront-app/src/components/js/EcOrdersList.js

```diff
@@ -33,7 +37,29 @@ export default {
   data () {
     return {
       updateInterval: null,
-      orders: []
+      orders: [],
+      pages: [],
+      items: [],
+      currentPage: null
+    }
+  },
+
+  computed: {
+    links () {
+      const first = [1, '...']
+      const last = ['...', this.pages.length]
+      let range = []
+
```

Comment:
```suggestion
    pageLinks () {
      const first = [1, '...']
      const last = ['...', this.pages.length]
      let range = []
```

`links` não é um nome genérico demais aqui? Considerando o contexto do componente...
Dentro de um componente "lista de pedidos" uma variável "link" não parece fazer referência a paginação...